canvas_modules/harness/src/client/components/sidepanel-properties.jsx

```diff
@@ -631,6 +656,41 @@ export default class SidePanelModal extends React.Component {
 			</Button>
 		</div>);
 
+		const setHideEditButtonPropertyId = (
+			<div className="harness-sidepanel-children" id="sidepanel-properties-set-disable-edit-button-propertyid">
+				<TextInput
+					labelText="Set the propertyId for enabling/disabling hideEditButton"
+					id="harness-propertyId-hideEditButton"
+					placeholder='{ "name": "parameterName" }'
+					invalid={this.state.invalidSetHideEditButtonPropertyId}
+					invalidText="Please enter valid JSON"
+					onChange={ this.setHideEditButtonPropertyId }
+					helperText='PropertyId format: {"name": "unique_id_for_control"}'
+				/>
+			</div>
+		);
+
+		const setHideEditButtonEnabled = (
+			<div className="harness-sidepanel-children" id="sidepanel-properties-set-disable-edit-button-disabled">
+				<Toggle
+					id="harness-sidepanel-setHideEditButtonEnabled-toggle"
+					labelText="Set hideEditButton disabled for the propertyId entered above"
```

Comment:
instead of `disabled` this should be `hide`

Reply:
updated